Others/horner.java

```diff
@@ -0,0 +1,32 @@
+// Java program for implementation of Horner Method 
+// for Polynomial Evaluation
+import java.io.*;
+ 
+class HornerPolynomial
+{
+    // Function that returns value of poly[0]x(n-1) +
+    // poly[1]x(n-2) + .. + poly[n-1]
+    static int horner(int poly[], int n, int x)
```

Comment:
The argument ```n``` is unnecessary. Because you can get the length of the array by ```length``` property.